build/zip.py

```diff
@@ -19,10 +19,11 @@
   'pyproto',
 ]
 
-def skip_path(dep):
+def skip_path(dep, dist_zip, target_cpu):
   should_skip = (
     any(dep.startswith(path) for path in PATHS_TO_SKIP) or
-    any(dep.endswith(ext) for ext in EXTENSIONS_TO_SKIP))
+    any(dep.endswith(ext) for ext in EXTENSIONS_TO_SKIP) or
+    'arm' in target_cpu and dist_zip == 'mksnapshot.zip' and dep == 'snapshot_blob.bin')
```

Comment:
please add more parentheses to make the semantics of this clear

also, how come `snapshot_blob.bin` isn't needed in the `mksnapshot.zip` on arm?

Reply:
snapshot_blob.bin gets generated by running mksnapshot but in this case mksnapshot can't run because its an arm binary built on an x64 system.  Since mksnapshot doesn't run in this case, snapshot_blob.bin doesn't exist.  It is in the list of dependencies passed in because v8_context_generator needs it to run but it can be generated by the user by running mksnapshot on their arm hardware.

Reply:
Please add a comment to the code explaining this :)

Reply:
@nornagon done.

Reply:
"can be generated"—does the user _have_ to generate that .bin in order to successfully use mksnapshot?